Admit.java

```diff
@@ -0,0 +1,136 @@
+ *
+ * No methods should have more than 15 lines of code minus curly braces or blank lines.
+ * One Scanner object.
+ * No for loops.  If else if okay.
+ */
+public class Admit {
+    public static void main(String[] args) {
+        introduction();
+        Scanner console = new Scanner(System.in);
+        applicantComparison(console);
+    }
+
+    /* @fn introduction()
+     * Prints the introduction message with directions in the console.
+     */
+    public static void introduction() {
+        System.out.println("This program compares two applicants to");
+        System.out.println("determine which one seems like the stronger");
+        System.out.println("applicant. For each candidate I will need");
+        System.out.println("either SAT or ACT scores plus a weighted GPA.");
+        System.out.println();
+    }
+
+    /* @fn applicantComparison(console)
+     * Solicits input for two applicants, prints their scores, then prints who is better candidate.
+     *
+     * @param console - allows collection of input from user
+     */
+    public static void applicantComparison(Scanner console) {
+        System.out.println("Information for applicant #1:");
+        double applicant1 = applicantDataRequest(console);
+        System.out.println("Information for applicant #2:");
+        double applicant2 = applicantDataRequest(console);
+        System.out.println("First applicant overall score = " + roundTo1Decimal(applicant1));
+        System.out.println("Second applicant overall score = " + roundTo1Decimal(applicant2));
+        if (applicant1 > applicant2) {
+            System.out.println("The first applicant seems to be better");
+        } else if (applicant1 < applicant2) {
+            System.out.println("The second applicant seems to be better.");
+        } else System.out.println("The two applicants seem to be equal.");
+    }
+
+    /* @fn applicantDataRequest(console)
+     * Collects input for one applicant and returns their overall score.
+     *
+     * @param console - allows collection of input from user
+     * @return value of their overall comparative score by adding examScore and gpaScore.
+     */
+    public static double applicantDataRequest(Scanner console) {
+        System.out.println("Do you have 1) SAT scores or 2) ACT scores?");
+        int examType = console.nextInt();
+        double examScore;
+        if (examType == 1) {
+            examScore = satScoreCalc(console);
+        } else examScore = actScoreCalc(console);
+        System.out.println("Exam score = " + roundTo1Decimal(examScore));
+        double gpaScore = gpaScoreCalc(console);
+        System.out.println("GPA Score = " + roundTo1Decimal(gpaScore));
+        System.out.println();
+        return examScore + gpaScore;
+    }
+
+    /* @fn satScoreCalc(console)
+     * Collects input for one applicant and calculates SAT exam score value.
+     *
+     * @param console - allows collection of input from user
```

Comment:
you just commented on what a console is, devs should know that, either leave blank or like 'used to get SAT scores from user' (something more useful)